dom0/sd-dom0-files.sls

```diff
@@ -79,6 +79,17 @@ dom0-update-securedrop-script-cron:
     - name: /etc/cron.daily/securedrop-update-cron
     - target: /usr/bin/securedrop-update
 
+# Copy uptime notification script into cron.hourly
+dom0-uptime-notify-securedrop-script-cron:
+  file.managed:
+    - name: /etc/cron.hourly/securedrop-uptime-notify
+    - source: salt://securedrop-uptime-notify
+    - user: root
+    - group: root
+    - mode: 755
+    - require:
+      - file: dom0-securedrop-icon
```

Comment:
why are you requiring the icon here?

Reply:
Ah! This is a relic from earlier approach using `notify-send`. Removing―